app/utils/CheckYourAnswersHelper.scala

```diff
@@ -39,6 +39,15 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
         getTrusteeFirstLastName(index))
   }
 
+  def trusteeAUKCitizen(index : Int): Option[AnswerRow] = userAnswers.get(TrusteeAUKCitizenPage(index)) map {
+    x =>
+      AnswerRow(
+        "trusteeAUKCitizen.checkYourAnswersLabel",
```

Comment:
This will have to accept the Name of the trustee it is referring to. Richy has implemented this for another page.